src/pages/explore/BiddersPage/BiddersList.tsx

```diff
@@ -10,8 +10,9 @@ import { humanizeBidKind } from '../../../helpers/humanize'
 import { truncateMiddle } from '../../../helpers/truncate'
 import { useKusama } from '../../../kusama'
 
-type Props = { bids: Vec<PalletSocietyBid> | [], activeAccount: accountType; handleResult: any }
+type Props = { bids: Vec<PalletSocietyBid>, activeAccount: accountType; handleResult: any }
 
+// TODO: move this to a `components` directory to follow the convention of other pages 
```

Comment:
I added this to-do instead of fixing it rightaway because diffs usually get messy when we move files and make changes to their content at the same time.